### PR TITLE
Feature/fix hide/show header

### PR DESCRIPTION
**Pull request by TatianaLopaeva**
Changes have been made for a mobile device:
- Added hide header when scroll down and show header when scroll up.
- Added full page scroll.
- The article does not have its own scroll bar.

### CODE REVIEW

File: products/ASC.Files/Client/src/components/pages/Home/Section/Body/index.js [hunk 1]
```diff
@@ -1856,7 +1875,7 @@ class SectionBodyContent extends React.Component {
 
     const { authorType, search, withSubfolders, filterType } = filter;
     const isFiltered = authorType || search || !withSubfolders || filterType;
-
+    console.log();
```
Comment (AlexeySafronov) [inline products/ASC.Files/Client/src/components/pages/Home/Section/Body/index.js:1878]: Useless console

File: web/ASC.Web.Common/src/components/Layout/index.js [hunk 2]
```diff
@@ -1,43 +1,218 @@
...
+const { size, isSmallTablet } = utils.device;
+
+const StyledContainer = styled.div`
+  width: 100%;
+  height: ${(props) =>
+    (props.isTabletView || isMobile) && !isFirefox
+      ? `${props.contentHeight}px`
+      : "100vh"};
+
+  /*#desktopScroll {
```
Comment (AlexeySafronov) [inline web/ASC.Web.Common/src/components/Layout/index.js:30]: Remove commented code

File: web/ASC.Web.Common/src/components/PageLayout/sub-components/article-pin-panel.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
-import { utils, Icons, Text } from "asc-web-components";
+import { utils, Icons, Text, Box } from "asc-web-components";
```
Comment (AlexeySafronov) [inline web/ASC.Web.Common/src/components/PageLayout/sub-components/article-pin-panel.js:4]: Box import is useless

File: products/ASC.Files/Client/src/components/pages/Home/Section/Body/index.js (hunk 1 above)
Comment (TatianaLopaeva) [reply products/ASC.Files/Client/src/components/pages/Home/Section/Body/index.js:1878]: Deleted

File: web/ASC.Web.Common/src/components/Layout/index.js (hunk 2 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Common/src/components/Layout/index.js]: Deleted.

File: web/ASC.Web.Common/src/components/PageLayout/sub-components/article-pin-panel.js (hunk 3 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Common/src/components/PageLayout/sub-components/article-pin-panel.js]: Deleted it.

**Review by AlexeySafronov — APPROVED** [review]

**Pull request merged**